license.txt

```diff
@@ -0,0 +1,22 @@
+/**
+  * The MIT License (MIT)
```

Comment:
AFAIK this project is Apache 2.0 license, so we would need to modify this accordingly.